Localization/Translations/Professions.lua

```diff
@@ -170,6 +170,18 @@ local professionLocals = {
         ["esES"] = "Equitación",
         ["frFR"] = "Monte",
     },
+    ["Inscription"] = {
+        ["ptBR"] = "Escrivania",
+        ["ruRU"] = "Надпись",
```

Comment:
@Iowerth suggested this change

```suggestion
        ["ruRU"] = "Начертание",
```